src/pages/about/aboutHelper.js

```diff
@@ -386,6 +386,69 @@ export const frontEnd = {
 			gitHub: "",
 			portfolio: "",
 			linkedIn: "",
+		},
```

Comment:
Add extra white space at top and bottom of this group that has been added. Add a TODO comment that lets us know that this group can be deleted when cohorts are implemented.